tests/basictests.cpp

```diff
@@ -233,6 +233,17 @@ namespace document_tests {
     }
     return true;
   }
+  bool padded_with_open_bracket() {
+    std::cout << __func__ << std::endl;
+    simdjson::dom::parser parser;
+    // This is an invalid document padded with open braces.
+    auto error1 = parser.parse("[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[", 2, false).error();
```

Comment:
I thought of a consistent way to test that this is happening: sanitize should reveal the error. Also, I *think* it will repro even if your document is parse("[[               ", 1, false). BTW, why did you extend the number of brackets? Is it a different repro case at all, or just to have parity with the next one?

Reply:
"BTW, why did you extend the number of brackets? Is it a different repro case at all, or just to have parity with the next one?" I think I copied and pasted this particular line straight out of your branch, the one you had experimented with.